pkg/etcd/etcd.go

```diff
@@ -65,7 +70,18 @@ type GetterSetter interface {
 // Returns:
 // 	This puts an *etcd.Client into the context.
 func CreateClient(c cookoo.Context, p *cookoo.Params) (interface{}, cookoo.Interrupt) {
-	url := p.Get("url", "http://localhost:4001").(string)
+	host := os.Getenv(hostEnvVar)
+	port, err := strconv.Atoi(os.Getenv(portEnvVar))
```

Comment:
This is not required as we are already doing it here 
https://github.com/deis/builder/blob/master/pkg/routes.go#L45